index.js

```diff
@@ -0,0 +1,74 @@
+const IncomingWebhook = require('@slack/client').IncomingWebhook;
+const humanizeDuration = require('humanize-duration');
+const config = require('./config.json');
+
+module.exports.webhook = new IncomingWebhook(config.SLACK_WEBHOOK_URL);
+module.exports.status = config.GC_SLACK_STATUS;
+
+// subscribe is the main function called by GCF.
+module.exports.subscribe = (event, callback) => {
+  let build = module.exports.eventToBuild(event.data.data);
+
+  // Skip if the current status is not in the status list.
+  let status = module.exports.status || ['SUCCESS', 'FAILURE', 'INTERNAL_ERROR', 'TIMEOUT'];
+  if (status.indexOf(build.status) == -1) {
+    return callback();
+  }
+
+  // Send message to slack.
+  let message = module.exports.createSlackMessage(build);
+  module.exports.webhook.send(message, (err, res) => {
+    if (err) console.log('Error:', err);
+    callback(err);
+  });
+};
+
+// eventToBuild transforms pubsub event message to a build object.
+module.exports.eventToBuild = data => {
```

Comment:
nit: I'd recommend putting () around data, s/data/(data) - that would be more consistent with Google's recommendations. (applies below as well)